test/environmentSettings.ts

```diff
@@ -6,11 +6,13 @@
  */
 
 import { join } from 'path';
+import * as utilities from './utilities';
 
 export class EnvironmentSettings {
   private static _instance: EnvironmentSettings;
 
   private _vscodeVersion = 'stable'; //  or 'insiders' or '1.77.3'
+  private _sfdxProjectHome = join(__dirname, '..', '..');
```

Comment:
I think `join(__dirname, '..', '..', 'apex-perf-project');` would be more appropriate for `_sfdxProjectHome`.

Also, I think `_sfdxTestProjectLocation` or `_apexPerfProjectLocation` would be a more appropriate name.

Reply:
I am consider these changes not for the purpose of satisfying my needs, but for the needs that follow.
Just trying to satisfy the need for an alternative project.